rdrf/rdrf/templates/rdrf_cdes/patient_info_component.html

```diff
@@ -22,6 +22,9 @@ <h3 style="margin:5px"><span class="label label-default">{% trans patient_type %
                     {% if pi.guid %}
                         &nbsp;|&nbsp;<strong>GUID:</strong>&nbsp;{{ pi.guid}}
                     {% endif %}
+                    {% if show_archive_button %}
+                    | <a class="link-danger" href="{{ archive_patient_url }}" data-bs-toggle="modal" data-bs-target="#archive_modal" target="_blank">Delete</a>
```

Comment:
Not sure about this. The link feels out of place next to the other information.

Some alternative places to try moving the link to:
1. A column in the patient list
2. A section on the demographics page
3. A modal dialog connected to a link on the patient name, that groups similar actions on the patient as a whole

What do you think?

Reply:
This is another tricky one.

1. I like this idea but the Patient List page is already quite busy, and on smaller screens it would be out of view.
2. This could work. Just worried about it being a bit hidden away as we are taking an option that's currently available on every patient form page and just making it available from one.
3. Not sure about this option. Use of modal dialogs should be minimised. It makes sense in the context of confirming irreversible (or hard to reverse) actions e.g. Confirming the deletion of an item, but not so much in the context of displaying options.
Also, the patient name is already link on form pages to navigate back to the Demographics page. But in saying that, I'm not really sure of the benefit of linking to the Demographics page from the patient name considering the option is available in the side bar.

Further thoughts extending on Option #3, we could turn the user name into a dropdown button with further options available from there.
E.g. 
![image](https://user-images.githubusercontent.com/60163872/188020319-ee331051-58a7-4c37-bf5b-a60c8cc4f303.png)

or we could use a single tab with a dropdown:
![image](https://user-images.githubusercontent.com/60163872/188022188-c26ab036-1a77-42f2-9398-97370297e5d0.png)

Reply:
**NB** We will need to update the patient list configuration in the MND registry to include new column 'actions'
e.g. from:
```
 "patient_list": {"columns": ["full_name", "date_of_birth", {"code": {"label": "Gender"}}, "last_updated_overall_at", "living_status", "modules"], "facets": ["working_groups", {"living_status": {"default": "Alive"}}]}
```
Needs to be updated to:
```
 "patient_list": {"columns": ["full_name", "date_of_birth", {"code": {"label": "Gender"}}, "last_updated_overall_at", "living_status", "modules", "actions"], "facets": ["working_groups", {"living_status": {"default": "Alive"}}]}
```

Moved the delete to the Patient Listing page as we had discussed:
![image](https://user-images.githubusercontent.com/60163872/189575538-c7458b72-2128-452a-936c-f2a5ebb36f82.png)

Delete modal now displays the patient's name: 
![image](https://user-images.githubusercontent.com/60163872/189575584-c71e5e99-4d03-4271-bc96-6e721d12d06b.png)

And no more delete button or link in the patient banner: 
![image](https://user-images.githubusercontent.com/60163872/189575627-0ab5089a-7f4a-4e31-a566-3beb936d0dc8.png)

Reply:
@ScriptSmith  This is now ready for your review whenever you have time (low priority)